templates/prescription/general_list.html

```diff
@@ -83,6 +83,33 @@
     }
 }
 
+function deleteDrug(d) {
+    let isPromise = top.jsFetchGlobals('e_rx');
+    isPromise.then(
+        msg => {
+            alert('msg.e_rx.deleteWarn');
+            if (choice == true) {
```

Comment:
also need this:
```
top.restoreSession();
```

Reply:
added. For my reference sake, when does top.restoreSession(); get added. I know the purpose. I just don't know when to use it. I don't have to be used all the time. So, under which conditions should it be added/used?

Reply:
What is the status on this feature?

Reply:
@juggernautsei I've been conflicted on keeping this feature. I removed in dialog but still like idea of keeping for globals in JS.
I'd remove and add back Brady's solution as more appropriate for a smarty form. Sorry!
`let msg = {/literal}{xlj t='Do you really want to delete?'}{literal};`

Reply:
@sjpadgett I am unclear how far back to go. Do I need to undo the promise.then?